app/src/main/java/org/simple/clinic/patient/AgeFuzzerModule.kt

```diff
@@ -0,0 +1,11 @@
+package org.simple.clinic.patient
+
+import dagger.Module
+import dagger.Provides
+
+@Module
+open class AgeFuzzerModule {
```

Comment:
Do we need a separate module just for `AgeFuzzer`?
Shouldn't this be part of `PatientSearchModule` ?

Reply:
The problem is that `PatientRepository` is responsible for calculating the fuzziness, and it is in the `AppScope`. This comes back to why the test setup for this is not as good as I'd have preferred.

Ideally, the fuzzer should be provided to the call instead of being injected into the repository, but that would require a lot of change.